Form/Type/ColorSelectorType.php

```diff
@@ -33,22 +33,41 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
      */
     public function configureOptions(OptionsResolver $resolver)
     {
-        $resolver->setDefaults(array(
-            'choices' => Colors::getAll(),
-            'translation_domain' => 'SonataCoreBundle',
-            'preferred_choices' => array(
-                Colors::BLACK,
-                Colors::BLUE,
-                Colors::GRAY,
-                Colors::GREEN,
-                Colors::ORANGE,
-                Colors::PINK,
-                Colors::PURPLE,
-                Colors::RED,
-                Colors::WHITE,
-                Colors::YELLOW,
-            ),
-        ));
+        if(method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
```

Comment:
Please add the missing space after `if` and `git commit --amend`, this should fix the failing build.